src/react-navigation.js

```diff
@@ -88,4 +88,13 @@ module.exports = {
   get withNavigation() {
     return require('./views/withNavigation').default;
   },
+
+  // Redux integration
+  get constructReactNavigationReduxMiddleware() {
```

Comment:
naming of "construct" sorta conflicts with the "create" naming used elsewhere in the library